test/balances.test.mjs

```diff
@@ -99,4 +109,17 @@ describe('aos Balances', async () => {
       ),
     );
   });
+
+  it('should set the logo of the ant', async () => {
+    const logo = 'my-logo-'.padEnd(43, '0');
+    const result = await handle({
+      Tags: [
+        { name: 'Action', value: 'Set-Logo' },
+        { name: 'Logo', value: logo },
+      ],
+    });
+
+    const info = await getInfo(result.Memory);
+    assert(info.Logo === logo, 'Failed to set logo');
+  });
```

Comment:
do we have 100% coverage on handlers? at the very least happy paths?

Reply:
We do now [883021e](https://github.com/ar-io/ar-io-ant-process/pull/32/commits/883021e6b1cb4e0efa8bf8136f79b78b94ebd80c)

Reply:
how about unit tests?